app/scripts/models/auth_brokers/mixins/channel.js

```diff
@@ -17,11 +17,48 @@ define([
   'underscore',
   'lib/promise'
 ], function (_, p) {
+  function doIt(strategy, channel, message, data) {
```

Comment:
![](https://psychoseamstress.files.wordpress.com/2015/02/arnold-do-it-now-mvpexchange-com.jpg)